packages/dd-trace/src/config.js

```diff
@@ -611,7 +611,7 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
     this.startupLogs = isTrue(DD_TRACE_STARTUP_LOGS)
     // Disabled for CI Visibility's agentless
     this.telemetry = {
-      enabled: DD_TRACE_EXPORTER !== 'datadog' && isTrue(DD_INSTRUMENTATION_TELEMETRY_ENABLED),
```

Comment:
Doesn't this have the opposite effect? Meaning that telemetry will actually always be off?

Reply:
how so? `DD_INSTRUMENTATION_TELEMETRY_ENABLED`'s default is `true` so this would also be true for ci vis.

Reply:
Won't that set it to `false` when `DD_TRACE_EXPORTER === 'datadog'` since it's an AND condition?

Reply:
mmm I'm _removing_ this boolean. The end state is 
```javascript
enabled: isTrue(DD_INSTRUMENTATION_TELEMETRY_ENABLED),
```
I might not be understanding completely 😅

Reply:
I completely misread that, don't mind me 😅 I was so focused on the line that I missed that it was _removed_.